core/invoker/src/main/scala/whisk/core/invoker/InvokerReactive.scala

```diff
@@ -126,6 +127,31 @@ class InvokerReactive(
       }
     }
 
+    // send activation metadata to kafka
+    val activation = Activation(
```

Comment:
Can you put this into a block in conjunction with the `EventMessage` below to prevent allocation if events are disabled?